src/results/new-skipped.js

```diff
@@ -0,0 +1,32 @@
+class NewSkipped {
+    constructor(runId, repository, maxNumberOpenedIssues, state) {
+        this.runId = runId;
+        this.repository = repository;
+        this.maxNumberOpenedIssues = maxNumberOpenedIssues;
+        this.state = state;
+    }
+
+    render() {
+        const slug = this.state.slug;
+        const orgName = this.repository.owner;
+        const repo = this.repository.repo;
+        const runId = this.runId;
+        const maxOpenedIssues = this.maxNumberOpenedIssues;
+        const component = `[${slug}](https://github.com/${orgName}/${repo}/actions/runs/${runId}#user-content-result-${slug})`;
```

Comment:
Keep in mind we have customers of self-hosted GitHub Enterprise.

Reply:
@osterman what would be the difference with enterprise?

Reply:
I made the urls relative. 
Is it fine?